docs/contributing/code.rst

```diff
@@ -134,7 +131,7 @@ django CMS you need to have the following dependencies installed:
     1. `Node <https://nodejs.org/>`_ version 0.12.7 (will install npm as well).
        We recommend using `NVM <https://github.com/creationix/nvm>`_ to get
        the correct version of Node.
-    2. `Globally installed gulp <https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md#1-install-gulp-globally>`_
+    2. `gulp <https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md>`_
```

Comment:
On https://github.com/gulpjs/gulp/blob/master/docs/API.md and http://gulpjs.com ("gulp is a toolkit") it's mostly not.